### DISCUSSION TITLE
Run on GPU

### DISCUSSION THREAD

**Discussion by MohammadMoradpoor** [PaddlePaddle/PaddleOCR#12894] [Q&A]
我想在GPU上运行这个软件包，但它没有任何输出，也没有任何错误。我正在使用Ubuntu 22.04，CUDA版本为11.7，cuDNN版本为8.6.0。请告诉我为了在GPU上运行此代码，我应该使用哪些版本。
@alldbi

**Comment by g39088902** [comment]
按照以下步骤排查：
1、如果你使用了vgpu，确保你正确安装了vgpu驱动和激活码。否则进入下一步。
2、输入nvidia-smi查看当前驱动所支持的cuda版本。确保驱动所支持的cuda版本与你安装的cuda版本一致。
3、尝试更换其他版本的驱动和CUDA，例如525驱动+CUDA 12.0。
4、尝试降级paddlepaddle版本。
